tests/unit/runtime/zero/test_zero_context.py

```diff
@@ -296,6 +295,8 @@ def forward(self, input):
         assert len(self._external_params) == 0
         assert len(self.container._external_params) == 1
         assert len(self.container.dangler._external_params) == 0
+        assert id(self.container.dangler.d_linear.bias
```

Comment:
may be adding a comment here from your explanation earlier - as it'd be difficult to remember why the test checks for `id` equivalency.